docs/src/tutorials/conic/min_ellipse.jl

```diff
@@ -5,110 +5,140 @@
 
 # # Minimal ellipses
 
-# This example comes from section 8.4.1 of the book *Convex Optimization* (Boyd
-# and Vandenberghe, 2004).
+# This example comes from section 8.4.1 of the book *Convex Optimization* by [Boyd
+# and Vandenberghe (2004)](https://web.stanford.edu/~boyd/cvxbook/).
 
-# Given a set of `m` ellipses `E(A, b, c) = { x : x' A x + 2 b' x + c ≤ 0 }`, we
-# find the ellipse of smallest area that encloses the given ellipses.
+# Given a set of ``m`` ellipses of the form 
+# ```math
+# E(A, b, c) = \{ x : x' A x + 2 b' x + c \leq 0 \},
+# ```
+# we find the ellipse of smallest area that encloses the given ellipses.
 
-# It is convenient to parameterize as the ellipse `{ x : || Px + q || ≤ 1 }`.
+# It is convenient to parameterize as the minimal enclosing ellipse as
+# ```math
+# \{ x : || Px + q || \leq 1 \}.
+# ```
 
-# Then the optimal `P` and `q` are given by the convex program
+# Then the optimal ``P`` and ``q`` are given by the convex semidefinite program
+# ```math
+# \begin{aligned}
+# \text{maximize } & \quad \log(\det(P))  \\
 
```

Comment:
```suggestion
```

Okay. Debugging these latex failures is nigh-on impossible. But it's usually because of empty lines like this. there's an open issue to parse the latex log for errors, but yeah. not fun. Going from `.jl` to `.md` to `LaTeX` means it could be any number of reasons :(

Reply:
Just pushed a formatted version. Fingers crossed.